util/common/agent_util_unix.go

```diff
@@ -41,10 +43,14 @@ func CopyFile(pathIn string, pathOut string) {
 	}
 
 	log.Printf("File %s abs path %s", pathIn, pathInAbs)
-	out, err := exec.Command("bash", "-c", "sudo cp "+pathInAbs+" "+pathOut).Output()
-
+	cmd := exec.Command("bash", "-c", "sudo cp "+pathInAbs+" "+pathOut)
```

Comment:
Why are we changing this?

Reply:
It gives us more information about the execution. So instead of saying the error is "exit status 1" it will actually give an error like "config file doesn't exist". I think it is a valid improvement